pkg/operator/ceohelpers/unsupported_override.go

```diff
@@ -0,0 +1,26 @@
+package ceohelpers
+
+import (
+	"bytes"
+	"encoding/json"
+	operatorv1 "github.com/openshift/api/operator/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/klog"
+)
+
+// IsUnsupportedUnsafeEtcd returns true if
+// useUnsupportedUnsafeNonHANonProductionUnstableEtcd key is set
+// to any parsable value
+func IsUnsupportedUnsafeEtcd(spec *operatorv1.StaticPodOperatorSpec) (bool, error) {
+	unsupportedConfig := map[string]interface{}{}
+	if err := json.NewDecoder(bytes.NewBuffer(spec.UnsupportedConfigOverrides.Raw)).Decode(&unsupportedConfig); err != nil {
+		klog.V(4).Infof("decode of unsupported config failed with error: %v", err)
+		return false, err
+	}
+
+	_, found, err := unstructured.NestedFieldNoCopy(unsupportedConfig, "useUnsupportedUnsafeNonHANonProductionUnstableEtcd")
```

Comment:
cant useUnsupportedUnsafeNonHANonProductionUnstableEtcd also be false?

Reply:
yeah but why? we will direct users to not use a bool value and use a `""` now they presence of key will determine the config

Reply:
ok I missed the comment